pkg/container/dict/dict.go

```diff
@@ -0,0 +1,197 @@
+package dict
```

Comment:
Please prepend apache 2.0 license header to every .go source file.

You can refer to any existing *.go

Reply:
Okay, done.